README.md

```diff
@@ -27,3 +28,4 @@ docker compose up
 - Hadoop HDFS: http://hadoop-master1.orb.local:9870
 - Hadoop YARN: http://hadoop-master1.orb.local:8088
 - Hadoop MapReduce JobHistory: http://hadoop-master1.orb.local:19888
+- Ranger Admin: http://hadoop-master1.orb.local:6080
```

Comment:
oh, I forget to provide this link https://docs.orbstack.dev/docker/domains

it is out-of-box on macOS with OrbStack instead of Docker Desktop